.github/workflows/test.yml

```
@@ -39,14 +40,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Set up Java 11
-        if: matrix.os == 'ubuntu-latest' && matrix.node-version == 12 && github.ref != 'refs/heads/master'
+      - name: Set up Java ${{ matrix.java-version }}
+        if: matrix.node-version == 12
```

```suggestion
        if: matrix.node-version == 12 || github.ref == 'refs/heads/master'
```

Do not run on master, cause we have a seperate workflow there

---

Should be  `&& github.ref != 'refs/heads/master'` then, should it not?

---

Sure.